tests/conftest.py

```diff
@@ -28,6 +29,9 @@ def __init__(self, name, parent, path):
         self.path = path
 
     def runtest(self):
+        readme_path = os.path.join(self.path.dirname, 'README.rst')
```

Comment:
Probably best to require `README*`, not .rst. People don't tend to write .rst READMEs these days:

```python
readmes = glob.glob(os.path.join(self.path.dirname, 'README*'))
if not readmes:
    raise ...

Reply:
Fair enough. Though sphinx also works with .md files via [recommonmark](https://github.com/jupyterhub/jupyterhub/blob/0.7.2/docs/source/conf.py#L55). I'm always hesitant to make people write new .rst documents.